src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -536,25 +535,25 @@ public bool IsBrokerInstalledAndInvokable(AuthorityType authorityType)
         {
             if (!DesktopOsHelper.IsWin10OrServerEquivalent())
             {
-                _logger?.Warning("[WAM Broker] Not a supported operating system. WAM broker is not available. ");
+                _logger?.Warning("[RuntimeBroker] Not a supported operating system. WAM broker is not available. ");
```

Comment:
FWIW, we hide both WAM and Runtime concepts in MSAL Python, so the parameter and logs all mention "broker" only.

This PR LGTM.

Reply:
Earlier I found it a bit difficult sometimes to figure out from logs which WAM implementation was used. Now that WAM runtime is used everywhere except UWP, it's less of an issue.